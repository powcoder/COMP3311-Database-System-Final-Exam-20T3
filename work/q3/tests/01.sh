https://powcoder.com
代写代考加微信 powcoder
Assignment Project Exam Help
Add WeChat powcoder
https://powcoder.com
代写代考加微信 powcoder
Assignment Project Exam Help
Add WeChat powcoder
psql footy -c 'select * from q3 order by team'
