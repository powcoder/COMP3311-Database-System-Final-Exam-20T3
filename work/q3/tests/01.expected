  team  | nplayers 
--------+----------
 Brazil |       14
(1 row)
